topcoffea/modules/createJSON.py

```diff
@@ -151,7 +156,8 @@ def main():
     if args.includeLheWgts:
         sampdic['nSumOfLheWeights'] = n_sum_of_lhe_weights
 
-    outname = sample
+    if sample != '':
+        outname = sample
```

Comment:
Else what happens? I.e. what will outcome be in the case when sample is ''?

Reply:
Then the outname would be the specified outname from the argparse. If both have been left empty, then the output would be '.json'
Could we raise an exception if both are empty?

Reply:
Yes I think it would be good to raise an exception instead of writing to ".json"